client.go

```diff
@@ -21,7 +23,18 @@ type Client struct {
 }
 
 func NewClient() (*Client, error) {
-	conn, err := net.Dial("unix", fmt.Sprint("/run/user/", os.Getuid(), "/pulse/native"))
+	socketNetwork := "unix"
+	socketPath := fmt.Sprint("/run/user/", os.Getuid(), "/pulse/native")
+	if pathRaw, ok := os.LookupEnv("PULSE_SERVER"); ok {
+		path := strings.SplitN(pathRaw, ":", 2)
+		if len(path) != 2 {
```

Comment:
https://www.freedesktop.org/wiki/Software/PulseAudio/Documentation/User/ServerStrings/
If the network type is not specified, it should be handled as unix socket.

Reply:
fixed by 9737046